qa/tasks/cephfs_test_runner.py

```diff
@@ -133,7 +133,9 @@ def task(ctx, config):
 
     # Mount objects, sorted by ID
     if hasattr(ctx, 'mounts'):
-        mounts = [v for k, v in sorted(ctx.mounts.items(), lambda a, b: cmp(a[0], b[0]))]
+        mounts = [v for k, v in sorted(
+            ctx.mounts.items(),
+            lambda a, b: (a[0] > b[0]) - (a[0] < b[0]))]
```

Comment:
Use key for py2/py3 compatibility:
```
mounts = [v for k, v in sorted(ctx.mount.items(), key=lambda _: _[0])]
```

Reply:
I would prefer todo that in another PR. I'm sure there are plenty of problems with python3. This is just a first step to catch the errors that flake8 (run on python3) can catch for us.

Reply:
This is very cheap change. It is much easier to change it here now, just with git commit --amend, than rerun the tests later.

Reply:
It's unrelated to the commit messages and unrelated to this PR. This is just a single py2/py3 compat change and there are maybe 100 more needed. The question for me is: How can we

- fix all theses py2/py3 compat issues and
- make sure that no new regressions for python3 are introduced

If you really want me to do this change, I can do it. But I'm not in favor of it.

Reply:
from my point it is directly related to this commit message and PR, because it removes cmp usage the same way you did it

Reply:
IMHO, this is not in the scope of pep8 compliance. probably we can leave the py3 compatible change to another commit or PR?

Reply:
This is exactly about py3 compatibility, since this change required for python3 flake8, if it is not a scope py3 than let's run py27 flake8 and it will not complain against this line and this change is not required then :-)

Reply:
i see. `run_tox.sh` uses both python2 and python3 envs when running the tox test.

Reply:
FYI - I already removed the 2nd commit which included this change. So this is no longer relevant. But my plan is to propose the flake8-python3 run (including the fixes) in another PR and then this gets relevant again.